external/updateDocumentation.m

```diff
@@ -0,0 +1,32 @@
+function updateDocumentation()
+% updateDocumentation
+%	Updates HTML documentation files for all RAVEN functions. It should
+%	only be used when preparing `devel` branch for a new RAVEN release
+%
+%	Usage: updateDocumentation()
+
+%Get the RAVEN path
+[ST, I]=dbstack('-completenames');
+[ravenDir,~,~]=fileparts(fileparts(ST(I).file));
+
```

Comment:
COBRA toolbox also redistribute `m2html`, but without all template files. If the COBRA path is higher up the MATLAB path than the RAVEN path, then the COBRA version is used. With this suggestion, the RAVEN `m2html` is used by moving that specific path to the top of the MATLAB path list.
```suggestion
%Make sure that RAVEN-provided m2html is used
path(fullfile(ravenDir,'software','m2html'),path);
```